spec/controllers/reports_controller_spec.rb

```diff
@@ -61,7 +61,7 @@
         expect(last_event.params).
           to match(
             request_id: info_request.id,
-            editor: user,
+            editor: user.url_name,
```

Comment:
Think we should include a bit more information for these instance objects, like: 

```
editor: { type: 'User', id: 1, url_name: 'foo' },
authority: { type: 'PublicBody', id: 1, title: 'Ministry of ...', version: 4 }
```

Reply:
> `editor: { type: 'User', id: 1, url_name: 'foo' }`

I was thinking that we'd have an explicit `Event#editor` relation rather than storing that in params.

> `authority: { type: 'PublicBody', id: 1, title: 'Ministry of ...', version: 4 }`

+1, nice. Certainly the record class and ID, as then we can constantise and find the record to render out more information. FWIW, BC stores this as an [`Event::Details`](https://github.com/garethrees/on_writing_software_well/blob/master/bc3/app/models/recording/eventable.rb#L16-L19) associated with the `Event`.